packages/terra-application-utility/src/utility/_UtilityMenu.scss

```diff
@@ -0,0 +1,135 @@
+:local {
+  .header-utility-menu {
+    background-color: var(--terra-application-header-utility-menu-background-color);
+    font-family: inherit;
+    width: 100%;
+    /* stylelint-disable selector-max-compound-selectors*/
+    .header {
+      display: flex;
+      flex-direction: column;
+      flex-shrink: 0;
+      font-size: var(--terra-application-header-utility-menu-header-font-size, 1rem);
+      font-weight: var(--terra-application-header-utility-menu-header-font-weight, normal);
+      line-height: var(--terra-application-header-utility-menu-header-line-height, 1.2);
+      margin-top: var(--terra-application-header-utility-header-margin-top, 0.7143rem);
+
+      /* stylelint-disable max-nesting-depth */
```

Comment:
You should flatten your selectors instead of disabling the linter. Turn this rule off when you need like 1 extra level of nesting.